cnv_somatic_panel_workflow.wdl

```diff
@@ -26,6 +26,8 @@
 #
 #############
 
+import "../cnv_common_tasks.wdl" as CNVTasks
+
 import "https://raw.githubusercontent.com/gatk-workflows/gatk4-somatic-cnvs/1.3.0/cnv_common_tasks.wdl" as CNVTasks
```

Comment:
Reminder to update path to use next version of workflow release before merging to master